lib/elixir/lib/macro.ex

```diff
@@ -370,6 +370,30 @@ defmodule Macro do
       iex> Macro.escape({:unquote, [], [1]}, unquote: true)
       1
 
+  ## Comparison to `Kernel.quote/2`
+
+  The `escape/2` function is sometimes confused with `Kernel.SpecialForms.quote/2`,
+  because the above examples behave the same with both. The key difference is
+  best illustrated when the value to escape is stored in a variable.
+
+      iex> Macro.escape({:a, :b, :c})
+      {:{}, [], [:a, :b, :c]}
+
+      iex> quote do: {:a, :b, :c}
+      {:{}, [], [:a, :b, :c]}
+
+      iex> value = {:a, :b, :c}
+      iex> Macro.escape(value)
```

Comment:
`...>` instead of `iex>`

Reply:
TIL! Is there a functional difference, or does it just better communicate that the second line is a continuation?

Reply:
No difference, just communication.

Reply:
But in this case `iex>` is better because it does not span over multiple lines.